profiler/src/ProfilerEngine/Datadog.Profiler.Native.Windows/EtwEventsManager.cpp

```diff
@@ -323,9 +323,10 @@ bool EtwEventsManager::Start()
     buffer << NamedPipePrefix;
     buffer << pid;
     std::string pipeName = buffer.str();
+
     Log::Info("Exposing ", pipeName);
 
-    _eventsHandler = std::make_unique<EtwEventsHandler>(_logger.get(), this);
+    _eventsHandler = std::make_unique<EtwEventsHandler>(_logger.get(), this, (FILE*)nullptr);
```

Comment:
no need to cast 🤔 ? is it for documentation purpose ?
Nit: use `static_cast<FILE*>(..)` instead ;)

Reply:
ok : let's remove it